plugins/inputs/opcua_listener/sample.conf

```
@@ -187,6 +187,12 @@
   #       deadband_type = "Absolute"
   #       deadband_value = 0.0
   #
+
+  ## Include additional Fields in each metric
+  # [inputs.opcua.additional_fields]
+    ## OPC-UA DataType
+    # include_datatype = true
```

The commented out value should represent the default value
```suggestion
    # include_datatype = false
```